blur-overview@nailfarmer.nailfarmer.com/files/blur-overview@nailfarmer.nailfarmer.com/po/fi.po

```diff
@@ -1,4 +1,4 @@
-# SOME DESCRIPTIVE TITLE.
+ttävyy# SOME DESCRIPTIVE TITLE.
```

Comment:
```suggestion
# SOME DESCRIPTIVE TITLE.
```

Reply:
@mahtiankka If it's okay, I'm going to go ahead and make this change to fix the error. The rest of the updates look fine.